java/arcs/core/host/ArcHostContext.arcs

```diff
@@ -0,0 +1,27 @@
+// These schema definitions are used to store a de-normalized version of ArcHostContext via ArcHostContextParticle
+
+schema HandleConnection
+  arcId: Text
+  particleName: Text
+  handleName: Text
+  storageKey: Text
+  mode: Text
```

Comment:
Documentation for these -- what are possible modes, types, TTL? Feels like we need to document this somewhere, either here or where the recipe/entity is used. Same below re: particleState, arcState, etc. Maybe it's already documented in a data class somewhere I'm just not seeing as it's out of this PR.

Reply:
These correspond to existing Kotlin classes, I can add doc references